### DISCUSSION TITLE
Community Meeting 2022-07-13

### DISCUSSION THREAD

**Discussion by JamieMagee** [microsoft/component-detection#134] [Announcements]
📆 Wednesday 13th July 2022
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220615T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda

- Add documentation on how to bump syft
- Speak to SBOM tool team to figure out the differences and if we can combine projects or work more closely together
- .NET 6 upgrade
- OSV integration
	- Reach out after .NET 6 upgrade and SBOM output merged. Speak to Oliver Chang in OpenSSF slack.

## Attendees

- @JamieMagee (Microsoft)
- @cobya (Microsoft)
- @laurajjiang (Microsoft)

## Recording

https://user-images.githubusercontent.com/1358764/178801793-c041cf5d-dbdc-4431-b7a6-4739386b3a83.mp4

**Comment by JamieMagee** [comment]
- [Issues opened since last meeting](https://github.com/microsoft/component-detection/issues?q=is%3Aissue+created%3A%3E2022-06-15)
- [PRs opened since last meeting](https://github.com/microsoft/component-detection/pulls?q=is%3Apr+created%3A%3E2022-06-15)

**Comment by cobya** [comment]
Agenda item which will be discussed as this will increment a major version of component detection #107
